test/libsolidity/syntaxTests/types/magic_block_basefee_error.sol

```diff
@@ -2,14 +2,8 @@ contract C {
     function f() public view returns (uint) {
         return block.basefee;
     }
-    function g() public view returns (uint ret) {
-        assembly {
-            ret := basefee()
-        }
-    }
 }
 // ====
 // EVMVersion: =berlin
```

Comment:
Starting from here you can then make it ``<=``, though :-).

Reply:
I'll change this once everything gets merged :)

Reply:
Actually, will do this now.

Reply:
fixed.